stable/airflow/templates/_helpers.tpl

```diff
@@ -84,14 +84,9 @@ Create the name for the airflow secret.
 Map environment vars to secrets
 */}}
 {{- define "airflow.mapenvsecrets" -}}
-    {{- $secretName := .Release.Name | trunc 63 | trimSuffix "-" }}
-    {{- $mapping := .Values.airflow.defaultSecretsMapping }}
-    {{- if .Values.existingAirflowSecret }}
-      {{- $secretName := .Values.existingAirflowSecret }}
-      {{- if .Values.airflow.secretsMapping }}
-        {{- $mapping := .Values.airflow.secretsMapping }}
```

Comment:
This variable is scoped only within the `if` block, meaning it's new value is lost further down the function.

Reply:
I wasted a good bit of time hitting this today as well, thanks for the fix! I temporarily hacked around it in my values file by using `defaultSecretsMapping`.